.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+rustdocflags = [ "--html-in-header", "./resources/docs-header.html" ]
```

Comment:
I'm a bit confused about how this interoperates with `rustdoc-args` and why it is in a separate file. In particular, if it's a global setting, why does it need to be repeated in `rustdoc-args`?

Reply:
I am not 100% sure myself, it's what they tell you to do in the example repo (https://github.com/victe/rust-latex-doc-minimal-example?tab=readme-ov-file) so I cargo-culted it in here. I think this entry in `.cargo/config.toml` is for local builds (you can also use env vars or pass the arg on the command line manually) and the `[metdata]` entry is for docs.rs to make sure it is passed when the crate is published.

Ideally docs.rs would also look here for config values I guess?